modules/versioning.nix

```diff
@@ -5,7 +5,19 @@ let
   version = config.nix-bitcoin.configVersion;
 
   # Sorted by increasing version numbers
-  changes = [
+  changes = let
+    mkOnionServiceChange = service: {
+      version = "0.0.29";
```

Comment:
This should be `0.0.30` as `v0.0.29` was released in the meantime and this PR will be released with `v0.0.30`.

Reply:
Fixed.